ibis/pandas/core.py

```diff
@@ -196,14 +220,21 @@ def execute_with_scope(expr, scope, aggcontext=None, clients=None, **kwargs):
 
 @trace
 def execute_until_in_scope(
-    expr, scope, aggcontext=None, clients=None, post_execute_=None, **kwargs
+    expr,
```

Comment:
at some point we should try to type more args

Reply:
Make sense, just add a typing.py for annotation purpose of those args.